tachyon/c/math/elliptic_curves/msm/msm_gpu_unittest.cc

```diff
@@ -44,8 +44,10 @@ TEST_P(MSMGpuTest, MSMPoint2) {
   for (const VariableBaseMSMTestSet<bn254::G1AffinePoint>& t :
        this->test_sets_) {
     std::unique_ptr<tachyon_bn254_g1_jacobian> ret;
-    std::vector<Point2<BigInt<4>>> bases = base::CreateVector(
-        t.bases.size(), [&t](size_t i) { return t.bases[i].ToMontgomery(); });
+    std::vector<Point2<bn254::Fq>> bases =
+        base::CreateVector(t.bases.size(), [&t](size_t i) {
+          return Point2<bn254::Fq>(t.bases[i].x(), t.bases[i].y());
```

Comment:
Don't we need to handle the infinity case where x and y are all zero?

Reply:
We don't need checking here!